src/Meshes.jl

```diff
@@ -365,6 +365,7 @@ export
   TriRefinement,
   QuadRefinement,
   CatmullClark,
+  PTQ,
```

Comment:
We prefer more descriptive and explicit names in Julia. Is there a standard name for this method? Ideally we retain the name of the original paper that introduced it.

Reply:
It is called PTQ in several papers, and in CGAL.

Reply:
And this a subdivision algorithm of the so-called family of **loop** subdivision algorithms.

Reply:
> It is called PTQ in several papers, and in CGAL.

The R language calls many things, ABC, DET, PTYS, ..., but this is not necessarily a good practice. It makes things unsearchable and the Julia community is working really hard to avoid unnecessary acronyms everywhere. We should rename it to a more descriptive searchable name. Charles Loop's thesis describes a couple of algorithms in chapter 3, which one is the one in this PR? https://charlesloop.com/thesis.pdf

Reply:
I never talked about R here. I didn't see any name in the thesis. I first found the name *Primal Triangle Quadrisection* in the CGAL documentation, and googling this name yields some results.

Reply:
Maybe `TriQuad`? For "triangle quadrisection"?

Reply:
> I never talked about R here.

I know :) Just pointing out that acronyms are terrible choices and we should avoid whenever possible. We use them sometimes, but it is worth checking if there is a more descriptive name to (1) honor the author of the algorithm and (2) make the code more searchable.

> I didn't see any name in the thesis.

![image](https://user-images.githubusercontent.com/3345261/209481774-26e864a1-c791-4c3a-825e-6f603322ba0b.png)

His chapter 3 describes a couple of algorithms. You can see that he mentions Catmull-Clark there, which we already have implemented here. Maybe the algorithm you implemented is one of the other ones?

Reply:
> Maybe `TriQuad`? For "triangle quadrisection"?

I think it is a bit confusing. I read it as a refinement procedure that produces triangles and quadrangles, but that is not the case here.

Reply:
`TriQuadrisectionRefinement`?